lib/my_home_page.dart

```diff
@@ -361,12 +361,24 @@ class _MyHomePageState extends ConsumerState<MyHomePage> {
           serverUri: _webId?.split('profile')[0] ?? 'Not connected',
           displayText: _webId?.split('profile')[0] ?? 'Not connected',
           isClickable: _webId != null,
+          tooltip:
```

Comment:
If the default tooltips are not suitable then we might discuss changing the defaults?